packages/shared/components/inputs/index.js

```diff
@@ -2,6 +2,7 @@ export { default as AccountInput } from './AccountInput.svelte'
 export { default as AliasInput } from './AliasInput.svelte'
 export { default as AmountInput } from './AmountInput.svelte'
 export { default as AssetAmountInput } from './AssetAmountInput.svelte'
+export { default as AssetAmountInput2 } from './AssetAmountInput2.svelte'
```

Comment:
Token amount input? I think we need to more towards token instead of asset when the component is explicitly for a token